Contributing.md

```diff
@@ -0,0 +1,70 @@
+# To contribute to this repository, follow the steps below:
+
+## Step 1: Fork this repository
+
+●Click the "Fork" button at the top right corner of this repository's page.
+
+![Alt text](../../Users/Lenovo/Documents/fork.png)
+
+## Step 2:Clone the forked repository
+
+●Go to your GitHub account and open the forked repository.
+●Click the "Code" button and then the copy icon to copy the repository's URL.
+
+![Alt text](../../Users/Lenovo/Documents/code.png)
+
+●Open a terminal and run the following command:
+git clone "url you just copied"
+
+![Alt text](../../Users/Lenovo/Documents/copy.png)
+
+The command will look something like this :
+git clone https://github.com/{your github user name}/AI-Powered-News-Article-Classifier-with-History.git
+
+## Step 3:Navigate to the Repository
+
+●Open the terminal and change your directory to the cloned repository.
+
+cd AI-Powered-News-Article-Classifier-with-History/
```

Comment:
Can we have to different path command for backend and frontend folders

Reply:
So I have to add a separate path for the backend folder for the people who want to contribute to the backend and the same for the frontend part of it?

Reply:
Yes, U got me